### ISSUE TITLE
Update spec status and/or publish?

### ISSUE THREAD

**Issue by gkellogg** [w3c/json-ld-bp#43]
These document was never published, but the specStatus indicates its a WG-NOTE. This causes issues when people look at the ED and see a reference to a non-existing note.

IMO, the group clearly intended to publish this as a NOTE, and it has been referenced. It also gets confused with the former [CG note](https://json-ld.org/spec/latest/json-ld-api-best-practices/), which it obsoletes. IIRC, the WG is still capable of publishing this as a NOTE, and even with open issues, it should be done.

Noted in discussion on Framing: https://github.com/w3c/json-ld-framing/issues/133#issuecomment-1229728032 (w3c/json-ld-framing#133)..

In the interim, the specStatus should be `DNOTE` (Draft Group Note).

**Comment by gkellogg** [comment]
Note, I changed status to `DNOTE`, and even tried `ED` and `unofficial` (sorry, intended to do this in a PR); it still references the /TR version as last published. Not sure why it does that. @iherman?

**Comment by iherman** [comment]
To be honest, @gkellogg, I don't really know. What happens if you set the status to `base` and you do not set the short name?

**Comment by gkellogg** [comment]
ReSpec must be getting status from somewhere else. Commenting out the shortName doesn't change anything. Setting specStatus to "base" (locally) generates some errors:

> W3C Working Group documents can't use "base" for the [specStatus](https://respec.org/docs/#specStatus) configuration option.
> How to fix: Pleas see [specStatus](https://respec.org/docs/#specStatus) for appropriate status for W3C Working Group documents.
(plugin: "w3c/defaults")
> Documents with a status of "base" can't be published on the W3C's /TR/ (Technical Report) space.
> How to fix: Ask a W3C Team Member for a W3C URL where the report can be published and change [latestVersion](https://respec.org/docs/#latestVersion) to something else.
(plugin: "w3c/headers")

Note that `respecConfig` does not contain a `latestVersion`, either. Using "unofficial" just reports the issue about `latestVersion`. I believe that the correct status for this should be DNOTE, as it is, in fact, a Draft Group Note, but not sure what to do about the Latest published version, though.

Although I believe it's still find to use a shortName, it's pretty odd that removing it doesn't cause it to go away.

**Comment by iherman** [comment]
> Although I believe it's still find to use a shortName, it's pretty odd that removing it doesn't cause it to go away.

I agree. I would go as far as saying that, for documents marked as "undefined" or "base", this is a respec bug.

(I did play around with my local copy and it did not make me any wiser. The only hack I could see is to run a post generation script that removes that header.)

>I believe that the correct status for this should be DNOTE, as it is, in fact, a Draft Group Note, but not sure what to do about the Latest published version, though.

The only proper way ahead is for the WG to officially publish this document as a Draft Note. The WG can do this, and that would remove all ambiguities.

Note, b.t.w., that the [charter of the WG](https://www.w3.org/2020/08/json-ld-wg-charter.html) runs out... tomorrow! Something should be done about it...

Cc @plehegar

**Comment by gkellogg** [comment]
Ouch! I thought the group was chartered to go quite a bit longer as a maintenance group. It will certainly have things to do to respond to RDF-star changes, and the YAML-LD activity  (described in the charter) has really ramped up. Best if we can keep the group going then to establish a charter for a new group.

cc @BigBlueHat

**Comment by BigBlueHat** [comment]
The JSON-LD WG was rechartered on January 19th, 2023 until January 31, 2025. Consequently, we're in a good position to formalize the note as a publication.

**Comment by BigBlueHat** [comment]
Now that we're recharted, I believe `DNOTE` "Draft Group Note" (which is the current spec status) is correct. Closing this one, consequently.
